hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -1113,23 +1121,24 @@ object HoodieSparkSqlWriter {
                                     writerSchema: Schema,
                                     dataFileSchema: Schema,
                                     operation: WriteOperationType,
-                                    instantTime: String) = {
+                                    instantTime: String,
```

Comment:
is createHoodieRecordRdd static?
can we create a new class called HoodieCreateRecordUtils and move this entire method there. the main class (HoodieSparkSqlWriter) is becoming unmanageable.

Reply:
Done.